### DISCUSSION TITLE
Alternative to Raspberry

### DISCUSSION THREAD

**Discussion by bbassac** [probonopd/MiniDexed#175] [Q&A] [answered]
Hi, in my country, we can't find raspberry before 2023.
Is there a way to make work this solution on a Teensy  (3.5 + ) ? 
I got a I2c simple LCD too . is there any chance to make it works ok i should surrend ? 
I got lot of ideas for this synth (simple voice change from midi to bypass the button, direct synth control from midi controllers on keyboard ..... execpt if everything already exixts ^^)

**Comment by AlpesMachines** [comment, marked as answer]
https://www.parasitstudio.de

2*DX7 in a teensy + add-on (not tested)

**Reply by bbassac** [reply]
I didn't realized that there is no jack output on teensy ^^

**Reply by probonopd** [reply]
MicroDexed is a compatible 6-operator-FM-synth based on the Teensy(-3.6/-4.x) Microcontroller: https://codeberg.org/dcoredump/MicroDexed.

MiniDexed is using a lot of the same code, but can do more than one Tone Generator instance on more powerful Raspberry Pi boards. (Hence the name, "Mini" is a bit larger than "Micro".)

**Comment by positionhigh** [comment]
or go here:
https://codeberg.org/positionhigh/MicroDexed-touch
